include/quicr/client.h

```diff
@@ -54,6 +54,12 @@ namespace quicr {
          */
         Status Disconnect();
 
+        // --BEGIN CALLBACKS ----------------------------------------------------------------------------------
+        /** @name Client Calbacks
+         *      client transport specific callbacks
+         */
+        ///@{
```

Comment:
What is this notation?

Reply:
specific to doxygen... https://www.doxygen.nl/manual/grouping.html